Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -476,8 +475,7 @@ private SiteExtensionInfo GetPreInstalledExtension(string id)
                         }
                         else
                         {
-                            tracer.Trace("Generate xdt file to map application to run from wwwroot when request for /{0}", package.Identity.Id);
-                            GenerateRemapToWebRootApplicationHostXdt(installationDirectory, string.Format(CultureInfo.InvariantCulture, "/{0}", package.Identity.Id));
+                            tracer.Trace("No xdt file come with package, shouldn`t restart site when install finish.");
```

Comment:
remove "shouldn`t restart site when install finish." from trace, avoid causing miss-understanding.
itself won`t trigger restart, but if there is multiple site extension installation happen at the same time, others might trigger site restart.